admin/app/view_models/workarea/admin/commentable_view_model.rb

```diff
@@ -16,6 +16,10 @@ def comment_count
       def comments
         @comments ||= CommentViewModel.wrap(model.comments)
       end
+
+      def subscribed_users
+        @users ||= User.find(subscribed_user_ids)
```

Comment:
This will raise if one of those IDs can't be found (e.g. the user was deleted). It'd be better to use `in(id: subscribed_user_ids)` which would return whoever it can find without raising.